crowbar_framework/lib/crowbar/error/upgrade_status.rb

```diff
@@ -41,5 +41,11 @@ def initialize(step_name = "")
         super("Step #{step_name} cannot be finished, as it is not running")
       end
     end
+
+    class SaveUpgradeStatusError < StandardError
+      def initialize(msg)
+        super("Exception during saving the status file: #{msg}")
```

Comment:
hrm, i don't find it that strange. i just took it as it is. btw you introduced it here https://github.com/crowbar/crowbar-core/commit/814b2372120a4237bf1cbb3040230f5fb574fff4 ;-)